pages/clubs/Running.jsx

```diff
@@ -9,39 +11,28 @@ var Running = React.createClass({
   contextTypes: {
     intl: React.PropTypes.object
   },
-  setData : function(data){
-    for(var i = 0; i < data.length; i++){
-      var categoryLabel = data[i].category.toLowerCase();
+  componentDidMount: function() {
+    this.props.teachAPI.getClubsGuides(function(err, res){
+      if(err) {
+        this.setState({error: err});
+        return;
+      }
 
-      categoryLabel = "guide_category_" + categoryLabel.split(' ').join('_');
+      var data = res.body;
 
-      var localizedCategory = this.context.intl.formatMessage({id: categoryLabel});
+      for(var i = 0; i < data.length; i++){
+        var categoryLabel = data[i].category.toLowerCase();
 
-      if(localizedCategory !== categoryLabel){
-        data[i].category = localizedCategory;
-      }
-    }
-    this.setState({guides : data });
-  },
-  fetchJSON : function(path,callback){
-    var httpRequest = new XMLHttpRequest();
+        categoryLabel = "guide_category_" + categoryLabel.split(' ').join('_');
 
-    httpRequest.onreadystatechange = function() {
-      if(httpRequest.readyState === 4) {
-        if(httpRequest.status === 200) {
-          var data = JSON.parse(httpRequest.responseText);
+        var localizedCategory = this.context.intl.formatMessage({id: categoryLabel});
 
-          if (typeof callback === "function") {
-            callback(data);
-          }
+        if(localizedCategory !== categoryLabel){
+          data[i].category = localizedCategory;
         }
       }
-    };
-    httpRequest.open('GET', path);
-    httpRequest.send();
-  },
-  componentDidMount: function() {
-    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+      this.setState({guides : data });
+    }.bind(this));
```

Comment:
we support ES6 syntax, so if you need `bind`, you don't need bind, you need an ES6 arrow function =)